core/sdk-core/src/main/java/software/amazon/awssdk/core/rules/EndpointResult.java

```diff
@@ -23,7 +23,7 @@
 import software.amazon.awssdk.protocols.jsoncore.JsonNode;
 
 @SdkInternalApi
-public final class Endpoint {
+public final class EndpointResult {
```

Comment:
Should we use `EndpointResponse` to be consistent with other response POJOs? (We may have discussed this already, ignore this comment if this is what we have already decided)

Reply:
Yes we did. I don't really like that idea since it deviates from the SEP but doesn't really provide much value.